components/SettingsControls/src/Dependencies.props

```diff
@@ -0,0 +1,31 @@
+<!--
+    WinUI 2 under UWP uses TargetFramework uap10.0.*
+    WinUI 3 under WinAppSdk uses TargetFramework net6.0-windows10.*
+    However, under Uno-powered platforms, both WinUI 2 and 3 can share the same TargetFramework.
+    
+    MSBuild doesn't play nicely with this out of the box, so we've made it easy for you.
+
+    For .NET Standard packages, you can use the Nuget Package Manager in Visual Studio.
+    For UWP / WinAppSDK / Uno packages, place the package references here.
+-->
+<Project>
+  <!-- WinUI 2 / UWP -->
+  <ItemGroup Condition="'$(IsUwp)' == 'true'">
+    <PackageReference Include="CommunityToolkit.Uwp.Triggers" Version="8.0.0-beta.1"/>
+  </ItemGroup>
+
+  <!-- WinUI 2 / Uno -->
+  <ItemGroup Condition="'$(IsUno)' == 'true' AND '$(WinUIMajorVersion)' == '2'">
+    <PackageReference Include="CommunityToolkit.Uwp.Triggers" Version="8.0.0-beta.1"/>
+  </ItemGroup>
+
+  <!-- WinUI 3 / WinAppSdk -->
+  <ItemGroup Condition="'$(IsWinAppSdk)' == 'true'">
+    <PackageReference Include="CommunityToolkit.WinUI.Triggers" Version="8.0.0-beta.1"/>
+  </ItemGroup>
+
+  <!-- WinUI 3 / Uno -->
+  <ItemGroup Condition="'$(IsUno)' == 'true' AND '$(WinUIMajorVersion)' == '3'">
+    <PackageReference Include="CommunityToolkit.WinUI.Triggers" Version="8.0.0-beta.1"/>
+  </ItemGroup>
```

Comment:
Ah, this is the CI issue, you don't need a package reference anymore, just need to add a project reference in the csproj to the Triggers project.

(Don't add a thing in the root props file like was done elsewhere, just use the path in the project reference in the csproj. Though we had an example somewhere, but can't find it atm)